deploy/helm/grafana-operator/templates/deployment.yaml

```
@@ -21,9 +19,7 @@ spec:
       {{- end }}
       labels:
         {{- include "grafana-operator.selectorLabels" . | nindent 8 }}
```

`grafana-operator.selectorLabels` don't include `.Values.additionalLabels`, so removing the `with` block is a breaking change; since this change landed, it's not possible to set additional pod labels.

I believe this can be easily fixed by including `grafana-operator.labels` instead:
```suggestion
        {{- include "grafana-operator.labels" . | nindent 8 }}
```